app/lib/renderer/episode.rb

```diff
@@ -12,5 +12,21 @@ def render
       attach_images
       render_markdown
     end
+
+    # Extract vtt file
+    def generate_vtt_text
```

Comment:
I don't see whether this method is called? Should it be in `render`?

Reply:
Yes, it should be in render.